test2code/src/main/kotlin/com/epam/drill/test2code/coverage/CoverageSender.kt

```diff
@@ -15,49 +15,90 @@
  */
 package com.epam.drill.test2code.coverage
 
+import com.epam.drill.plugins.test2code.common.api.*
+import com.epam.drill.test2code.*
+import com.github.luben.zstd.*
 import kotlinx.coroutines.*
-import mu.KotlinLogging
-import java.util.concurrent.Executors
-import kotlin.coroutines.CoroutineContext
+import kotlinx.serialization.protobuf.*
+import mu.*
+import java.math.*
+import java.util.*
+import java.util.concurrent.*
 
 interface CoverageSender {
-    fun setSendingHandler(handler: SendingHandler)
     fun startSendingCoverage()
     fun stopSendingCoverage()
 }
 
+private val COVERAGE_RETENTION_LIMIT_BYTES = BigInteger.valueOf(JvmModuleConfiguration.getCoverageRetentionLimit())
```

Comment:
No reason to use the uppercase;
No reason to declare a private variable (its purpose is obvious from the use)